Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/urls.ts

```diff
@@ -242,6 +242,15 @@ const getSearchResultsUrl = ({ searchTerm, page, searchCategory }: IGetSearchRes
     return `${baseApiUrl}/Search/Get${searchCategory}SearchResults?${searchQuery}&${searchCategory}=true&${pageQuery}`;
 };
 
+// eslint-disable-next-line consistent-return
+const getContestBtnUrlString = (isCompete: boolean, id: number) => {
+    if (isCompete) {
+        return `/contests/${id}/compete`;
```

Comment:
ContestParticipationType

Reply:
removed the whole function, since its not necessery anymore, done